packages/react-flicking/src/react-flicking/index.tsx

```diff
@@ -162,6 +156,10 @@ export default class Flicking extends React.Component<IFlickingProps> {
   public getPrevIndex(physical: boolean) {
     return this.flicking.getPrevIndex(physical);
   }
+  public plugin(plugins: any[]) {
+    (this.flicking as any).plugin(plugins);
```

Comment:
@younkue Why is casting to `any` needed?

Reply:
@jongmoon has no  ```plugin``` method in `@egjs/flicking`